kernel/data-pipeline/scenario/cdc/client/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/client/handler/ExceptionHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.cdc.client.handler;
+
+import org.apache.shardingsphere.data.pipeline.cdc.client.util.ServerErrorResult;
+
+/**
+ * Exception handler.
+ */
+public interface ExceptionHandler {
+    
+    /**
+     * Handler server exception.
+     *
+     * @param result error result
+     */
+    void handlerServerException(ServerErrorResult result);
+    
+    /**
+     * Handler socket exception.
+     *
+     * @param throwable throwable
+     */
+    void handlerSocketException(Throwable throwable);
```

Comment:
`handlerServerException` could be `handleServerException`. And also `handlerSocketException`.